custom/shared/contexts/WaitingRoomProvider.js

```diff
@@ -31,6 +32,7 @@ export const WaitingRoomProvider = ({ children }) => {
         };
       })
     );
+    setMultipleWaiting(waiting.length > 1);
```

Comment:
Should we use the memo instead of updating the state every time?

Like this 

```
const multipleWaiting = useMemo(() => waiting.length > 1, [waiting])
```

Reply:
that makes sense to me 👍

Reply:
Thank you @harshithpabbati and @jessmitch42! I'm updating this.